internal/api/template_config_service.go

```diff
@@ -0,0 +1,89 @@
+package api
+
+import (
+	"context"
+
+	"github.com/DIMO-Network/vehicle-signal-decoding/internal/core/commands"
+	"github.com/DIMO-Network/vehicle-signal-decoding/internal/core/queries"
+
+	db "github.com/DIMO-Network/shared/db"
+	grpc "github.com/DIMO-Network/vehicle-signal-decoding/pkg/grpc"
+	"github.com/rs/zerolog"
+	"google.golang.org/protobuf/types/known/emptypb"
+)
+
+type TemplateConfigService struct {
+	grpc.TemplateConfigServiceServer
+	logger *zerolog.Logger
+	DBS    func() *db.ReaderWriter
```

Comment:
```suggestion
	dbs    func() *db.ReaderWriter
```
it isn't exported so better to be all lower

Reply:
done